app/dashboards/degree_dashboard.rb

```diff
@@ -42,13 +51,15 @@ class DegreeDashboard < Administrate::BaseDashboard
   # on the model's form (`new` and `edit`) pages.
   FORM_ATTRIBUTES = %i[
     theses
-    name
+    name_dspace
+    abbreviation
+    degree_type
   ].freeze
 
   # Overwrite this method to customize how degrees are displayed
   # across all pages of the admin dashboard.
   #
   def display_resource(degree)
-    degree.name
+    "#{degree.name_dw}"
```

Comment:
we shouldn't need to use string interpolation here. I know administrate does this a lot, but `degree.name_dw` is probably better.

Same feedback for all the dashboards being touched in this PR to skip interpolation unless we actually need it.